static/modules/buttons.js

```diff
@@ -108,6 +108,8 @@ export const buttons = {
   primary: new Button(" ", 4),
   /** @type {Button} */
   secondary: new Button("e", 5),
+  /** @type {Button} */
+  pause: new Button("p", 5),
```

Comment:
can we get this to work with the controller? change the second argument to whatever the "start" button number is